boto/s3/key.py

```diff
@@ -2530,7 +2530,7 @@ def set_remote_metadata(self, metadata_plus, metadata_minus, preserve_acl,
                             headers=headers)
 
     def select_object_content(self, data, headers=None):
-        md5 = compute_md5(BytesIO(data))
+        md5 = compute_md5(BytesIO(data.encode('utf-8')))
```

Comment:
Do we need this change?  I see this one in s3select.py.

`response = key.select_object_content(data.encode('utf-8'), headers=headers)`

Reply:
OK not needed!